observability/index.adoc

```diff
@@ -16,7 +16,7 @@ toc::[]
 * Network Observability
 * {PM-shortname-c}
 
-{ObservabilityLongName} connects open source observability tools and technologies to create a unified {ObservabilityShortName} solution. The components of {ObservabilityLongName} work together to help you collect, store, deliver, analyze, and visualize data.
+{ObservabilityLongName} connects open-source observability tools and technologies to create a unified {ObservabilityShortName} solution. The components of {ObservabilityLongName} work together to help you collect, store, deliver, analyze, and visualize data.
```

Comment:
Hello @Srivaralakshmi this change was made because the preferred form for adjective in this case is "open-source": https://www.ibm.com/docs/en/ibm-style?topic=word-usage: see **open-source** _adjective_  vs **open source** _adjective_
The supplementary style guide only refers to the case where the open source is a noun. Thank you!